### PR TITLE
Update instructions.html

### LINKED ISSUE #1072: Update instructions.html

### PR DESCRIPTION
**Pull request by mattwhisenhunt**
fixes #1072

### CODE REVIEW

**Comment by mattwhisenhunt** [comment]
@garrettjonesgoogle PTAL

File: instructions.html [hunk 1]
```diff
@@ -7,67 +7,21 @@ <h3>Browse Online</h3>
...
   Source</a></li>
 </ul>
 
 <h3>Checkout Instructions</h3>
 
-<p><b>Prerequisites:</b> install <a href="http://java.com">Java 6</a>, <a
-  href="http://mercurial.selenic.com/">Mercurial</a> and <a
+<p><b>Prerequisites:</b> install
+  <a href="http://www.oracle.com/technetwork/java/javase/downloads/index.html">
+    Java SE 8</a>, <a href="https://git-scm.com">Git</a> and <a
```
Comment (garrettjonesgoogle) [inline instructions.html]: Wait I thought Java 7 was the minimum...

Comment (mattwhisenhunt) [reply instructions.html]: It is. Maybe _Prerequisites_ is the wrong word to use here. I figured if a developer is following these instructions to install a jdk, directing them to download 8 from that page is a lot simpler than anything else.

Comment (garrettjonesgoogle) [reply instructions.html]: Well we should change the wording: Prerequisite is Java 7, but Java 8 is recommended

**Pull request closed without merging**